### PR TITLE
CC/ERT touchups, plus a small bugfix.

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
Touches up the CC agent/ERT area- still a WIP but removes some redundancy, tidies up the selection of gear and adds a bit of heavy equipment. Also adds a teleporter so agents can be deployed quickstyle 'without' having to be teleported by admins. Also adds some QOL stuff for them - also makes some effort to coralle ERT people away from areas that they shouldn't see and removes a couple of largely unused test-rooms in the main test room. I left that room largely intact for now but removed the excelsior/chunch shit.

Adds a paperbin/stamp for the specs shop.

Makes some tweaks to the CO/WO stamps to stand out a bit more from one another.

fixes the fucking missing food sprites god DAMMIT luxy

Adds a box of spare headsets

2 new sets of armor - a RIG and a set of regular armor. The former a slightly less slow shield RIG, the latter essentially equivalent to the mercenary vest/helmet. The armor is ERT only however the WO/CO get a version that has stats closer to their actual dept rigs.

armor/rig are both bay sprites that I'm hapy w/ the quality of. Thank u bay 

 ERT fellas now get a speshial ID card with an excellently coder-sprited 'H.C' on it. Instead of every agent getting a special unique ID with varying levels of access and requiring a lotta manual fucking with, instead they're all children of the agent ID and start with AA. This way, you can instantly imprint your details on, change your name on it to whatever your characters name is and boom. 

Further finishes mapping the ERT ready room and such. The area includes a teleporter, a guild prep area with supplies(Gee i wonder why this was necessary). 

The armory has been completely reworked, instead being mostly specialized powerful security gear and including far fewer redundant pieces - a rack absolutely laden with vouchers is included as well as a blackshield voucher vendor that was....missing. 

Militia/Marshal ERT get a new badge that's half silver half gold to go with the fact it's a dual mark that carries the authority of both the Provost and the Brigadier.

At Benl's suggestion adds a locked down room to hold the exo, the bolt button is in the relocated crayon magic test area where only an admin can press it.

Slight tweaks to CC( again)

Some new specialty perk/stat injectors as well as pre-filled boxes for quick equipping ERT/agents/etc etc with stats and perks - this ones still a WIP.

Fucking makes a prospector Underboss spawner for ease.

SRI Ethics committeemember as well.

Adds the flavor back to each access card for various HC agents- most have AA, some don't now(namely to SI med/sci and prospi agents)

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm
Comment (benj8560) [inline code/modules/clothing/suits/armor.dm:1]: ''This particular set bears a '' Mighta missed something there at the end

Comment (cdb-is-not-good) [reply code/modules/clothing/suits/armor.dm:1]: aghj

Comment (benj8560) [reply code/modules/clothing/suits/armor.dm:1]: WeH

File: code/game/objects/items/weapons/cards_ids_syndicate.dm [hunk 1]
```diff
@@ -205,3 +205,15 @@
...
+/obj/item/card/id/syndicate/ert
+	name = "operative id card"
+	desc = "A specialized ID for the appointed operatives of the high council."
+	registered_name = "Agent"
+	assignment = "Agent"
+	icon_state = "hc_all-access"
+	group = "golden"
+
+/obj/item/card/id/secert/New()
+	access = get_all_station_access()
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/cards_ids_syndicate.dm:218]: each ERT agent was giving different accesses that aline with their job not AA

Comment (cdb-is-not-good) [reply code/game/objects/items/weapons/cards_ids_syndicate.dm:218]: An access console *is* present in the CC area if we decide that a particular agent needs to have less access, but we should generally trust that ERTs(Who are often sent to assist in emergencies) can handle having emergency access imo

File: code/datums/outfits/jobs/ghostroles.dm [hunk 2]
```diff
@@ -41,7 +41,8 @@
 	backpack_contents = list(/obj/item/storage/firstaid/soteria = 1, /obj/item/modular_computer/tablet/moebius/preset = 1, /obj/item/gun/projectile/automatic/c20r/sci/preloaded = 1, \
 							/obj/item/gun_upgrade/trigger/dnalock = 1, /obj/item/gun_upgrade/muzzle/silencer = 1, /obj/item/bodybag/cryobag = 2, \
 							/obj/item/storage/firstaid/blackshield/large = 1, /obj/item/storage/firstaid/surgery/si = 1, /obj/item/roller/compact  = 1, /obj/item/device/defib_kit/compact/combat/adv/loaded = 1)
-	l_pocket = /obj/item/card/id/medical_command
+	id_slot = slot_wear_id
+	id_type =  /obj/item/card/id/syndicate/ert
```
Comment (Trilbyspaceclone) [inline code/datums/outfits/jobs/ghostroles.dm:45]: medical ERT shouldnt get AA

File: code/datums/outfits/jobs/ghostroles.dm [hunk 3]
```diff
@@ -58,22 +59,25 @@
 	head = /obj/item/clothing/head/soft/sarge2soft
 	back = /obj/item/storage/backpack/security
 	backpack_contents = list(/obj/item/ammo_magazine/rifle_75_drum = 3, /obj/item/ammo_magazine/rifle_75_drum/highvelocity = 2, /obj/item/ammo_casing/grenade/flash = 2, /obj/item/ammo_casing/grenade/blast = 2, /obj/item/modular_computer/tablet/lease/preset/command = 1 )
+	id_slot = slot_wear_id
+	id_type =  /obj/item/card/id/syndicate/ert
```
Comment (Trilbyspaceclone) [inline code/datums/outfits/jobs/ghostroles.dm:63]: marshal ERT already has an ID and its already AA

Comment (cdb-is-not-good) [reply code/datums/outfits/jobs/ghostroles.dm:63]: Please read why the IDs were changed in the changelog.

File: code/datums/outfits/jobs/ghostroles.dm [hunk 4]
```diff
@@ -58,22 +59,25 @@
...
 	glasses = /obj/item/clothing/glasses/sechud/tactical
 	shoes = /obj/item/clothing/shoes/swat
 	belt = /obj/item/storage/belt/security/tactical/shieldert
-	back = /obj/item/storage/backpack/duffelbag
-	backpack_contents = list(/obj/item/ammo_magazine/heavy_rifle_408_drum = 3, /obj/item/ammo_magazine/heavy_rifle_408_drum/slap = 2, /obj/item/ammo_magazine/heavy_rifle_408_drum/incend = 2, /obj/item/modular_computer/tablet/lease/preset/command = 1, /obj/item/storage/pouch/ammo = 1, /obj/item/clothing/suit/space/void/SCAF/blackshield = 1 )
-
+	back = /obj/item/storage/backpack/militia
+	backpack_contents = list(/obj/item/modular_computer/tablet/lease/preset/command = 1, /obj/item/storage/pouch/ammo = 1)
+	id_slot = slot_wear_id
+	id_type =  /obj/item/card/id/syndicate/ert
```
Comment (Trilbyspaceclone) [inline code/datums/outfits/jobs/ghostroles.dm:80]: This ERT already has an AA id card

File: code/datums/outfits/jobs/ghostroles.dm [hunk 5]
```diff
@@ -118,7 +123,8 @@
 							/obj/item/pen/multi = 1,
 							/obj/item/reagent_containers/glass/paint/red = 1, //For the papers, in case they want to make a paper red with red link to REALY show that they mean something
 							/obj/item/modular_computer/tablet/lease/preset/command = 1)
-	l_pocket = /obj/item/card/id/lss_command
+	id_slot = slot_wear_id
+	id_type =  /obj/item/card/id/syndicate/ert
```
Comment (Trilbyspaceclone) [inline code/datums/outfits/jobs/ghostroles.dm:127]: LSS ERT shouldnt get an AA access

File: code/game/objects/items/weapons/cards_ids_syndicate.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/cards_ids_syndicate.dm:218]: We dont trust agents tho, agents are people

File: code/modules/clothing/head/helmet.dm [hunk 6]
```diff
@@ -218,9 +218,9 @@
 
 /obj/item/clothing/head/helmet/swat
 	name = "\improper SWAT helmet"
-	desc = "These are often used by highly trained SWAT Members."
-	icon_state = "swat"
-	armor_list = list(melee = 80, bullet = 60, energy = 40, bomb = 50, bio = 10, rad = 0)
+	desc = "An oldie but a goodie. This helmet was often the last thing seen by those so foolish as to draw the attention of the Solarian Frontier enforcement corps. Judging by the dents, it didn't save its last wearer, but perhaps you'll fare better?"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/head/helmet.dm:221]: this means EVERY swat helm is dented and used. bit of an odd flavour

File: code/modules/clothing/spacesuits/rig/suits/combat.dm [hunk 7]
```diff
@@ -102,14 +102,6 @@
 	name = "blackshield hazard hood"
 	desc = "A hazard hood modded to help against CQC combat."
 	camera_networks = list(NETWORK_SECURITY)
-	armor_list = list(
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/rig/suits/combat.dm:105]: why are you nerfing bs hardsuit?

Comment (cdb-is-not-good) [reply code/modules/clothing/spacesuits/rig/suits/combat.dm:105]: there's 0 reason for the helmet to have extra armor on top of its already high armor.

File: code/modules/clothing/head/helmet.dm (hunk 6 above)
Comment (cdb-is-not-good) [reply code/modules/clothing/head/helmet.dm:221]: there are no swat helmets other than the ones i placed for this particular thing.

**Comment by Trilbyspaceclone** [comment]
im kinda against WO and CO having their own RIG suit

**Comment by cdb-is-not-good** [comment]
> im kinda against WO and CO having their own RIG suit

CO already has one. the stats are the same as that one, jsut a new sprite.

**Pull request merged**